Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -26,13 +26,30 @@ class AppStoreConnectService {
     }
 
     func readCertificate(serial: String) throws -> Certificate {
-        try ReadCertificateOperation(options: .init(serial: serial)).execute(with: requestor).await()
+        let sdkCertificate = try ReadCertificateOperation(options: .init(serial: serial))
+            .execute(with: requestor)
+            .await()
+
+        return Certificate(sdkCertificate)
     }
 
     func createCertificate(with options: CreateCertificateOptions) -> AnyPublisher<Certificate, Error> {
         CreateCertificateOperation(options: options).execute(with: requestor)
     }
 
+    func revokeCertificates(serials: [String]) throws -> [Void] {
```

Comment:
🤔 I don't think the void array output is useful:

```suggestion
    func revokeCertificates(serials: [String]) throws {
```

Reply:
@adamjcampbell will get `Unexpected non-void return value in void function` if remove `[Void]`